webviz_subsurface/plugins/_simulation_time_series/utils/create_vector_traces_utils.py

```diff
@@ -22,43 +22,50 @@
 
 
 def create_vector_observation_traces(
-    vector_observations: dict, color: str = "black", legend_group: Optional[str] = None
+    vector_observations: dict,
+    color: str = "black",
+    legend_group: Optional[str] = None,
+    show_legend: bool = False,
 ) -> List[dict]:
     """Create list of observations traces from vector observations
 
     `Input:`
     * vector_observations: dict - Dictionary with observation data for a vector
     * color: str - Color of observation traces for vector
-    * legend_group: Optional[str] - Name of legend group, added as legend group and name if provided
+    * legend_group: Optional[str] - Overwrite default legend group. Name of legend group, added as
+    legend group and name if provided.
+    * show_legend: bool - Show legend status for traces
+
 
     `Return:`
     List of marker traces for each observation for vector
     """
     observation_traces: List[dict] = []
 
+    _name = "Observation" if legend_group is None else "Observation: " + legend_group
+    _legend_group = "Observation" if legend_group is None else legend_group
+
     for observation in vector_observations.get("observations", []):
-        hovertext = observation.get("comment", "")
+        hovertext = observation.get("comment", None)
```

Comment:
```suggestion
        hovertext = observation.get("comment")
```
I think `None` is default?

Reply:
That's true!